### DISCUSSION TITLE
Nibe MyUplink

### DISCUSSION THREAD

**Discussion by Gooliga** [hp-net/homebridge-nibe#114] [Ideas]
Hi,
Any plans to add MyUplink support also? As Nibe is moving more towards Myuplink.com rather than old uplink.com it would be nice to add this support also.

**Comment by hpruszyn** [comment]
@Gooliga thank you very much for sharing this idea. 
I didn’t know about my uplink. However I did some research. 

First finding is that data is quite similar but organised in better way. In the end we still have parameter with id and value. 

Second finding - my heat pump(F1145 PC) is not supported by my uplink, I can’t connect my device. Nibe says on its webpage that my uplink is dedicated for new devices starting with S. In API documentation for developers, my uplink api is still marked as beta. 

So to sum it up: currently I will focus on old uplink api because it is the only one working for F devices (and others). However I will keep an eye on my uplink and if they will support older devices I will try to support that in the plugin. 
Or maybe someone will be interested in adding this integration to the plugin - I would like to have some company :)

**Comment by jmmo20** [comment]
I have a F2040 and recently NIBE got in touch telling me I could upgrade to myUplink. I did and then only after I realised I lost access to your plugin. did you get the same notification for your NIBE system?

**Comment by Gooliga** [comment]
for me I upgraded my “smart home” to Home Assistant.
It has Nibe support locally via ModBus over TCP/IP.
I am using SMO-40 + F2125. System is able to present approx 488 different configurable features (ie smart price adotption, additional hot water etc) + ~800 diagnostic items (ie energy log, temperatures etc etc). Also I can easily meter power consumption and effectiveness (COP).

https://www.home-assistant.io/integrations/nibe_heatpump￼
Nibe Heat Pump
home-assistant.io
